x/gamm/keeper/pool.go

```diff
@@ -104,98 +104,6 @@ func (k Keeper) DeletePool(ctx sdk.Context, poolId uint64) error {
 	return nil
 }
 
-// CleanupBalancerPool destructs a pool and refund all the assets according to
-// the shares held by the accounts. CleanupBalancerPool should not be called during
-// the chain execution time, as it iterates the entire account balances.
-// TODO: once SDK v0.46.0, use https://github.com/cosmos/cosmos-sdk/pull/9611
```

Comment:
Seems that there is some context here about bringing this back with SDK v0.46.0

I support removing dead code and reintroducing, once needed. However, we should probably document this in an issue, linking this PR

Reply:
I'd rather keep this personally